docs/v0.5/assets/css/sass.css

```diff
@@ -15,11 +15,11 @@
 
 pre.codeblock-pre { display: inline-flex; justify-content: left; text-align: left; align-items: left; align-content: left; flex-wrap: nowrap; flex-direction: row; background-color: #111; margin: 0; padding: 10px; border-radius: 0.3rem; width: 100%; }
 
-.codeblock { width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
+.codeblock { position: realative; width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
```

Comment:
```suggestion
.codeblock { position: relative; width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
```

Reply:
Since there's a spelling mistake here, i wondering if this is a required change?

Reply:
Nope that change wouldn't be necessary, adding/removing position: relative doesn't make an impact on the functionality. So we can just remove it entirely.